ApplicationExamples/N44/DAEMode_TestCases/N44_Base_Case_Systems/Generators/GenEventTest.mo

```diff
@@ -0,0 +1,97 @@
+within N44.DAEMode_TestCases.N44_Base_Case_Systems.Generators;
+model GenEventTest "Configuration of synchronous generator with regulators: GENSAL, HYGOV, SCRX, STAB2A;
+  Nordic 44 model: Buses 3115, 5300, 6100, 6700, 7100"
+  extends OpenIPSL.Electrical.Essentials.pfComponent;
+  OpenIPSL.Electrical.Machines.PSSE.GENSAL gENSAL(
+    Tppd0=0.05 "T''do (> 0)",
+    Tppq0=0.15 "T''qo (> 0)",
+    D=0 "Speed Damping",
+    Tpd0=6.4,
+    H=3.5,
+    Xd=1.14,
+    Xq=0.84,
+    Xpd=0.34,
+    Xppd=0.26,
+    Xppq=0.26,
+    Xl=0.2,
+    M_b=1200,
+    S10=0.1,
+    S12=0.3,
+    R_a=0,
+    V_b=V_b,
+    V_0=V_0,
+    angle_0=angle_0,
+    P_0=P_0,
+    Q_0=Q_0) annotation (Placement(transformation(extent={{-64,-24},{-2,40}})));
+  OpenIPSL.Electrical.Controls.PSSE.TG.HYGOV hYGOV(
+    R=0.06 "Permanent droop, p.u",
+    r=0.4 "Temporary droop, p.u",
```

Comment:
This is a strange place to add a description string. The description strings should be present in the original class not as part of the instance modifier.